pipelines/_ingredients/Conversions.yaml

```diff
@@ -26,12 +26,16 @@ tasks:
     class: lsst.pipe.tasks.postprocess.ConsolidateVisitSummaryTask
     config:
       connections.calexp: initial_pvi
+      connections.visitSummary: visitSummary
+      connections.visitSummarySchema: visitSummary_schema
```

Comment:
I don't understand why this change is needed, and there's no explanation in the commit message.

Reply:
I am defining all of the connections with their default names, before changing them to the new names in a later commit.